bin/peas_shell

```diff
@@ -354,7 +354,7 @@ class PanSensorShell(cmd.Cmd):
             sensor = getattr(self, sensor_name)
             try:
                 sensor.capture(store_result=True, send_message=True)
-            except Exception as e:
+            except Exception:
```

Comment:
Should we be figuring out what exceptions this might be?

Reply:
I've got a replacement coming for this code, so don't want to bother.